gallery/src/components/NxTabs/NxTabsModalExample.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel,
+  NxModal,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+
+import {faAngry} from '@fortawesome/free-solid-svg-icons';
+
+const NxTabsModalExample = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  const [activeTabId, setActiveTabId] = useState<number>(0);
+
+  function handleTabSelect(index: number) {
+    setActiveTabId(index);
+  }
+
+  return (
+    <>
+      <button onClick={() => setShowModal(true)} className="nx-btn">Open Modal</button>
+      { showModal &&
+        <NxModal id="nx-modal-simple-example" onClose={modalCloseHandler} className="nx-modal--wide">
+          <header className="nx-modal-header">
+            <h2 className="nx-h2">
+              <NxFontAwesomeIcon icon={faAngry} />
+              <span>Modal header</span>
+            </h2>
+          </header>
+          <div className="nx-modal-content">
+            <NxTabs activeTab={activeTabId} onTabSelect={handleTabSelect}>
+              <NxTabList aria-label="Simple Tabs">
```

Comment:
Does this aria label need to be updated?

Reply:
> Does this aria label need to be updated?

All of them updated here: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/3aee5949b382944a42188aace36ed9c5aada9ab3

Reply:
> All of them updated here: 3aee594

They all appear unchanged to me

Reply:
> They all appear unchanged to me

Seems now that you got to them at some point.